airbyte-webapp/src/services/connectorBuilder/ConnectorBuilderStateService.tsx

```diff
@@ -119,9 +120,9 @@ export const ConnectorBuilderFormStateProvider: React.FC<React.PropsWithChildren
       editorView !== "ui"
         ? jsonManifest
         : builderFormValues === lastValidBuilderFormValues
-        ? jsonManifest
+        ? derivedJsonManifest
```

Comment:
FYI I pushed this change because I noticed when testing this branch that the stream URL in the testing panel was not being updated when I would change the URL path of the stream.

After some digging, I figured out that this should be using the `derivedJsonManifest` in this case, because `jsonManifest` just contains the local storage JSON value of the yaml editor, which means it contained outdated info.

The only reason this was working on master is because we were doing the `merge()` call above (which was removed in this PR), which caused `builderFormValues === lastValidBuilderFormValues` to always evaluate to `false`. Now that the merge is removed, that evaluates to true when there are no errors, so it was returning the `jsonManifest` value which was outdated, resulting in weird behavior.